test/puppetlabs/puppetdb/command_test.clj

```diff
@@ -1276,11 +1325,11 @@
                         :version 3
                         :payload {:certname "bar.example.com"
                                   :producer_timestamp (now)}}}
-             {:certname "bar.example.com"
+             #_{:certname "bar.example.com"
```

Comment:
I'll revisit these two tests. I remember something about them not being valid anymore. I'll either confirm that for fix it.